packages/docs/src/locales/en/en.json

```diff
@@ -900,6 +900,7 @@
         "filterMethod": "A custom filtering function. The function takes the initial value of the currently checked cell (the `source` is a formal parameter) and should return a `boolean` value, indicating whether to include the row containing this cell or not",
         "sortBy": "Sets the column to sort by. Works through the `v-model`",
         "sortingOrder": "Sets the sorting order. Works through the `v-model`",
+        "sortingOptions": "An array from a set of values (`'asc'`, `'desc'`, `null`) that specifies the order and allowed sort directions.",
```

Comment:
extra `.` in the end ?

Reply:
After discussion, we decided to use `.` at the end of sentences in tables.